regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy

```diff
@@ -0,0 +1,91 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_backup_restore_bloom_filter", "backup_restore") {
+    String repoName = "test_backup_restore_bloom_filter_repo"
+    String dbName = "test_backup_restore_bloom_filter_db"
+    String tableName = "test_backup_restore_bloom_filter_table"
+
+    def syncer = getSyncer()
+    syncer.createS3Repository(repoName)
+
+    sql "CREATE DATABASE IF NOT EXISTS ${dbName}"
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableName}"
+    sql """
+        CREATE TABLE ${dbName}.${tableName} (
+            `sale_date` date NOT NULL,
+            `saler_id` int NOT NULL,
+            `category_id` int NOT NULL
```

Comment:
Exception:
java.sql.SQLException: errCode = 2, detailMessage = Syntax error in line 6:
        Duplicate KEY(`id`)
        ^
Encountered: DUPLICATE
Expected: COMMA